gadget/validate.go

```diff
@@ -358,3 +358,32 @@ func ValidateContent(info *Info, gadgetSnapRootDir string) error {
 	}
 	return nil
 }
+
+// checkVolumeHasAllKernelRefs ensures that the given
+func checkVolumetHasAllKernelRefs(pNew *LaidOutVolume, kernelInfo *kernel.Info) error {
```

Comment:
this is more  checkAllVolumeKernelRefsAreAvailable ? or canResolveAllVolumeKernelRefs ?

Reply:
I like "canResolveAllVolumeKernelRefs"